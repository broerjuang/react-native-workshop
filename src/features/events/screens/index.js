//@flow

export {default as EventsScreen} from './EventsScreen';
