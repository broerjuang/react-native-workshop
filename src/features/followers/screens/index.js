// @flow
export {default as FollowersScreen} from './FollowersScreen.js';
