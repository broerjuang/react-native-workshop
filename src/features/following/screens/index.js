// @flow
export {default as FollowingScreen} from './FollowingScreen.js';
