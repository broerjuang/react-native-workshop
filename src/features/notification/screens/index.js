//@flow
export {default as AllNotificationsScreen} from './AllNotificationsScreen';
export {default as ParticipatingScreen} from './ParticipatingScreen';
export {default as UnreadScreen} from './UnreadScreen';
