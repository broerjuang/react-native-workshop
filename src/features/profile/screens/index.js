export {default as ProfileScreen} from './ProfileScreen';
export {default as SettingScreen} from './SettingScreen';
