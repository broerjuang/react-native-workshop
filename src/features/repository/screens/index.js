// @flow

export {default as RepositoryScreen} from './RepositoryScreen';
export {default as RepositoryDetailScreen} from './RepositoryDetailScreen';
export {default as RepositoryFileListScreen} from './fileList';
