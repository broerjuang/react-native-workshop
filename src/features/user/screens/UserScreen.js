// @flow

import React, {Component} from 'react';
import {View, Text, TouchableOpacity, SafeAreaView, Image} from 'react-native';
import ParallaxScrollView from 'react-native-parallax-scroll-view';
import {connect} from 'react-redux';
import {MaterialIcons} from '@expo/vector-icons';
// import {createStore, applyMiddleware} from 'redux';

import DetailsGroup from '../../../global/core-ui/DetailsGroup';
import ParallaxButtons from '../../../global/core-ui/ParallaxButtons';
import RowWith3Column from '../../../global/core-ui/RowWith3Column';
import type {UserState} from '../reducers/userReducer';

type Props = {
  navigation: Object;
  handleAction: (action: Object) => void;
  userState: UserState;
};

type State = {};

export class UserScreen extends Component<Props, State> {
  render() {
    return (
      <SafeAreaView style={{flex: 1}}>
        <ParallaxScrollView
          styles={{height: 10000}}
          backgroundColor="#272727"
          contentBackgroundColor="white"
          parallaxHeaderHeight={300}
          stickyHeaderHeight={65}
          contentContainerStyle={styleParallax.contentStyle}
          renderStickyHeader={() => (
            <View style={styleParallax.stickyHeader}>
              <Text style={styleParallax.txtStickyHeader}>
                {this.props.userState.userLogin}
              </Text>
            </View>
          )}
          renderFixedHeader={() => (
            <View
              style={{
                position: 'absolute',
                right: 10,
                top: 25,
                height: 30,
                width: 30,
                alignItems: 'center',
                justifyContent: 'center',
              }}
            >
              <TouchableOpacity
                onPress={() => this.props.navigation.navigate('Setting')}
              >
                <MaterialIcons name="settings" size={30} color="white" />
              </TouchableOpacity>
            </View>
          )}
          renderForeground={() => (
            <View style={styleParallax.containerForeground}>
              <View style={styleParallax.containerProfilePicture}>
                <Image
                  style={styleParallax.imgProfilePicture}
                  source={{
                    uri: this.props.userState.userPicture,
                  }}
                />
              </View>
              <View style={styleParallax.containerFullName}>
                <Text style={styleParallax.txtFullName}>
                  {this.props.userState.userFullName}
                </Text>
              </View>
              <View style={styleParallax.containerUsername}>
                <Text style={styleParallax.txtUsername}>
                  {this.props.userState.userLogin}
                </Text>
              </View>
              <View style={styleParallax.containerButton}>
                <ParallaxButtons
                  name="Repositories"
                  value={this.props.userState.sumRepositories}
                  onPress={() =>
                    this.props.navigation.navigate('RepositoryScreen')
                  }
                />

                <ParallaxButtons
                  name="Stars"
                  value={this.props.userState.sumStars}
                  onPress={() =>
                    this.props.navigation.navigate('RepositoryScreen')
                  }
                />
                <ParallaxButtons
                  name="Followers"
                  value={this.props.userState.sumFollowers}
                  onPress={() => this.props.navigation.navigate('Followers')}
                />
                <ParallaxButtons
                  name="Following"
                  value={this.props.userState.sumFollowing}
                  onPress={() => this.props.navigation.navigate('Following')}
                />
              </View>
            </View>
          )}
        >
          <View style={styles.containerProfileDetails}>
            <DetailsGroup disabled={true} name="Bio">
              <RowWith3Column
                content={
                  this.props.userState.biography !== null ? (
                    <Text> {this.props.userState.biography}</Text>
                  ) : (
                    <Text>No Biography Found</Text>
                  )
                }
              />
            </DetailsGroup>
            <DetailsGroup name="Website">
              <RowWith3Column
                left={
                  <MaterialIcons
                    name="link"
                    size={35}
                    style={{
                      opacity: 0.5,
                    }}
                  />
                }
                content={
                  this.props.userState.website !== '' ? (
                    <Text> {this.props.userState.website}</Text>
                  ) : (
                    <Text> No Website Found</Text>
                  )
                }
              />
            </DetailsGroup>
            <DetailsGroup name="Organizations">
              {this.props.userState.organizations ? (
                this.props.userState.organizations.map((orgRow, i) => {
                  return (
                    <RowWith3Column
                      key={i}
                      isTouchable={true}
                      content={<Text>{orgRow.name}</Text>}
                    />
                  );
                })
              ) : (
                <RowWith3Column content={<Text>No Organization found</Text>} />
              )}
            </DetailsGroup>
          </View>
        </ParallaxScrollView>
      </SafeAreaView>
    );
  }
}

const styles = {
  containerProfileDetails: {
    flex: 1,
    flexDirection: 'column',
  },
};
const styleParallax = {
  contentStyle: {
    flex: 1,
  },
  containerForeground: {
    backgroundColor: '#272727',
    height: 300,
    flex: 1,
    paddingTop: 40,
    flexDirection: 'column',
  },
  stickyHeader: {
    backgroundColor: '#272727',
    justifyContent: 'center',
    alignItems: 'center',
    height: 80,
  },
  containerProfilePicture: {
    // flex: 1,
    height: 100,
    alignItems: 'center',
  },
  imgProfilePicture: {
    width: 90,
    height: 90,
  },
  containerFullName: {
    // flex: 1,
    height: 30,
    justifyContent: 'center',
    alignItems: 'center',
  },
  containerUsername: {
    // flex: 1,
    height: 20,
    justifyContent: 'center',
    alignItems: 'center',
  },
  containerButton: {
    flex: 1,
    flexDirection: 'row',
    justifyContent: 'space-between',
    paddingTop: 25,
    paddingBottom: 25,
    paddingLeft: 15,
    paddingRight: 15,
  },
  txtStickyHeader: {
    color: 'white',
    fontWeight: 'bold',
    fontSize: 20,
  },
  txtFullName: {
    color: 'white',
    fontWeight: 'bold',
    fontSize: 20,
  },
  txtUsername: {
    color: 'white',
    fontSize: 16,
  },
};

function mapStateToProps(state) {
  return {
    userState: state.userReducer,
  };
}

function mapDispatchToProps(dispatch) {
  return {
    handleAction: (action: Object) => dispatch(action),
  };
}

export default connect(
  mapStateToProps,
  mapDispatchToProps,
)(UserScreen);
