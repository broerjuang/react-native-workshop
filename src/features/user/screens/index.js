export {default as UserScreen} from './UserScreen';
