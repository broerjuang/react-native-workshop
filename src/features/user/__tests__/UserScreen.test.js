// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import {UserScreen} from '../screens/UserScreen';

describe('UserScreen', () => {
  it('should render UserScreen corectly', () => {
    let component = renderer.create(
      <UserScreen
        navigation={{navigate: (s) => s}}
        handleAction={() => {
          return;
        }}
        userState={{
          userLogin: 'asd',
          userFullName: 'asd',
          userPicture: 'asd',
          sumRepositories: 0,
          sumStars: 0,
          sumFollowers: 0,
          sumFollowing: 0,
          biography: 'asd',
          website: 'asd',
          organizations: [],
        }}
      />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
