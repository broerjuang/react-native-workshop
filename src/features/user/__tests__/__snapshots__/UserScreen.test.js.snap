// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserScreen should render UserScreen corectly 1`] = `
<RCTSafeAreaView
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    onLayout={[Function]}
    style={
      Array [
        undefined,
        Object {
          "backgroundColor": "transparent",
          "flex": 1,
        },
      ]
    }
  >
    <View
      collapsable={undefined}
      style={
        Object {
          "backgroundColor": "#272727",
          "height": 300,
          "opacity": 1,
          "overflow": "hidden",
          "position": "absolute",
          "top": 0,
          "transform": Array [
            Object {
              "translateY": 0,
            },
            Object {
              "scale": 1,
            },
          ],
          "width": 750,
        }
      }
    >
      <View>
        <View />
      </View>
    </View>
    <RCTScrollView
      collapsable={undefined}
      onChangeHeaderVisibility={[Function]}
      onScroll={[Function]}
      scrollEventThrottle={1}
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
      styles={
        Object {
          "height": 10000,
        }
      }
    >
      <View>
        <View
          style={
            Object {
              "backgroundColor": "transparent",
              "overflow": "hidden",
            }
          }
        >
          <View
            collapsable={undefined}
            style={
              Object {
                "backgroundColor": "transparent",
                "height": 300,
                "opacity": 1,
                "overflow": "hidden",
              }
            }
          >
            <View
              style={
                Object {
                  "height": 300,
                }
              }
            >
              <View
                style={
                  Object {
                    "backgroundColor": "#272727",
                    "flex": 1,
                    "flexDirection": "column",
                    "height": 300,
                    "paddingTop": 40,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "height": 100,
                    }
                  }
                >
                  <Image
                    source={
                      Object {
                        "uri": "asd",
                      }
                    }
                    style={
                      Object {
                        "height": 90,
                        "width": 90,
                      }
                    }
                  />
                </View>
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "height": 30,
                      "justifyContent": "center",
                    }
                  }
                >
                  <Text
                    accessible={true}
                    allowFontScaling={true}
                    ellipsizeMode="tail"
                    style={
                      Object {
                        "color": "white",
                        "fontSize": 20,
                        "fontWeight": "bold",
                      }
                    }
                  >
                    asd
                  </Text>
                </View>
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "height": 20,
                      "justifyContent": "center",
                    }
                  }
                >
                  <Text
                    accessible={true}
                    allowFontScaling={true}
                    ellipsizeMode="tail"
                    style={
                      Object {
                        "color": "white",
                        "fontSize": 16,
                      }
                    }
                  >
                    asd
                  </Text>
                </View>
                <View
                  style={
                    Object {
                      "flex": 1,
                      "flexDirection": "row",
                      "justifyContent": "space-between",
                      "paddingBottom": 25,
                      "paddingLeft": 15,
                      "paddingRight": 15,
                      "paddingTop": 25,
                    }
                  }
                >
                  <View
                    accessibilityComponentType={undefined}
                    accessibilityLabel={undefined}
                    accessibilityTraits={undefined}
                    accessible={true}
                    collapsable={undefined}
                    hasTVPreferredFocus={undefined}
                    hitSlop={undefined}
                    isTVSelectable={true}
                    nativeID={undefined}
                    onLayout={undefined}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "opacity": 1,
                      }
                    }
                    testID={undefined}
                    tvParallaxProperties={undefined}
                  >
                    <View
                      style={
                        Object {
                          "alignItems": "center",
                          "flex": 1,
                          "flexDirection": "column",
                          "justifyContent": "center",
                        }
                      }
                    >
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "white",
                            "fontSize": 24,
                            "fontWeight": "bold",
                          }
                        }
                      >
                         
                        0
                         
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "white",
                            "fontSize": 14,
                          }
                        }
                      >
                         
                        Repositories
                         
                      </Text>
                    </View>
                  </View>
                  <View
                    accessibilityComponentType={undefined}
                    accessibilityLabel={undefined}
                    accessibilityTraits={undefined}
                    accessible={true}
                    collapsable={undefined}
                    hasTVPreferredFocus={undefined}
                    hitSlop={undefined}
                    isTVSelectable={true}
                    nativeID={undefined}
                    onLayout={undefined}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "opacity": 1,
                      }
                    }
                    testID={undefined}
                    tvParallaxProperties={undefined}
                  >
                    <View
                      style={
                        Object {
                          "alignItems": "center",
                          "flex": 1,
                          "flexDirection": "column",
                          "justifyContent": "center",
                        }
                      }
                    >
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "white",
                            "fontSize": 24,
                            "fontWeight": "bold",
                          }
                        }
                      >
                         
                        0
                         
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "white",
                            "fontSize": 14,
                          }
                        }
                      >
                         
                        Stars
                         
                      </Text>
                    </View>
                  </View>
                  <View
                    accessibilityComponentType={undefined}
                    accessibilityLabel={undefined}
                    accessibilityTraits={undefined}
                    accessible={true}
                    collapsable={undefined}
                    hasTVPreferredFocus={undefined}
                    hitSlop={undefined}
                    isTVSelectable={true}
                    nativeID={undefined}
                    onLayout={undefined}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "opacity": 1,
                      }
                    }
                    testID={undefined}
                    tvParallaxProperties={undefined}
                  >
                    <View
                      style={
                        Object {
                          "alignItems": "center",
                          "flex": 1,
                          "flexDirection": "column",
                          "justifyContent": "center",
                        }
                      }
                    >
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "white",
                            "fontSize": 24,
                            "fontWeight": "bold",
                          }
                        }
                      >
                         
                        0
                         
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "white",
                            "fontSize": 14,
                          }
                        }
                      >
                         
                        Followers
                         
                      </Text>
                    </View>
                  </View>
                  <View
                    accessibilityComponentType={undefined}
                    accessibilityLabel={undefined}
                    accessibilityTraits={undefined}
                    accessible={true}
                    collapsable={undefined}
                    hasTVPreferredFocus={undefined}
                    hitSlop={undefined}
                    isTVSelectable={true}
                    nativeID={undefined}
                    onLayout={undefined}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "opacity": 1,
                      }
                    }
                    testID={undefined}
                    tvParallaxProperties={undefined}
                  >
                    <View
                      style={
                        Object {
                          "alignItems": "center",
                          "flex": 1,
                          "flexDirection": "column",
                          "justifyContent": "center",
                        }
                      }
                    >
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "white",
                            "fontSize": 24,
                            "fontWeight": "bold",
                          }
                        }
                      >
                         
                        0
                         
                      </Text>
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "white",
                            "fontSize": 14,
                          }
                        }
                      >
                         
                        Following
                         
                      </Text>
                    </View>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </View>
        <View
          onLayout={[Function]}
          style={
            Array [
              Array [
                Object {
                  "backgroundColor": "white",
                },
                Object {
                  "flex": 1,
                },
              ],
              Object {
                "minHeight": 0,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "display": "none",
              }
            }
          />
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "column",
              }
            }
          >
            <View>
              <View
                disabled={true}
                style={
                  Object {
                    "alignItems": "center",
                    "borderBottomColor": "grey",
                    "borderBottomWidth": 1,
                    "flexDirection": "row",
                    "justifyContent": "space-between",
                    "padding": 15,
                    "paddingTop": 30,
                  }
                }
              >
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "fontWeight": "bold",
                    }
                  }
                >
                  Bio
                </Text>
                <View>
                   
                   
                </View>
              </View>
              <View
                style={
                  Object {
                    "borderBottomColor": "grey",
                    "borderBottomWidth": 2,
                    "flex": 1,
                    "paddingBottom": 15,
                    "paddingTop": 15,
                  }
                }
              >
                <View
                  disabled={true}
                  style={
                    Object {
                      "justifyContent": "center",
                    }
                  }
                >
                  <View
                    accessibilityComponentType={undefined}
                    accessibilityLabel={undefined}
                    accessibilityTraits={undefined}
                    accessible={true}
                    collapsable={undefined}
                    hasTVPreferredFocus={undefined}
                    hitSlop={undefined}
                    isTVSelectable={true}
                    nativeID={undefined}
                    onLayout={undefined}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "opacity": 1,
                      }
                    }
                    testID={undefined}
                    tvParallaxProperties={undefined}
                  >
                    <View
                      style={
                        Array [
                          Object {
                            "alignItems": "center",
                            "backgroundColor": "white",
                            "flexDirection": "row",
                            "justifyContent": "center",
                          },
                          Array [],
                        ]
                      }
                    >
                      <View
                        style={
                          Object {
                            "alignItems": "center",
                            "justifyContent": "center",
                            "width": "15%",
                          }
                        }
                      />
                      <View
                        style={
                          Object {
                            "alignItems": "flex-start",
                            "flexWrap": "wrap",
                            "justifyContent": "center",
                            "padding": 10,
                            "width": "70%",
                          }
                        }
                      >
                        <Text
                          accessible={true}
                          allowFontScaling={true}
                          ellipsizeMode="tail"
                        >
                           
                          asd
                        </Text>
                      </View>
                      <View
                        style={
                          Object {
                            "alignItems": "center",
                            "justifyContent": "center",
                            "width": "15%",
                          }
                        }
                      />
                    </View>
                  </View>
                </View>
              </View>
            </View>
            <View>
              <View
                style={
                  Object {
                    "alignItems": "center",
                    "borderBottomColor": "grey",
                    "borderBottomWidth": 1,
                    "flexDirection": "row",
                    "justifyContent": "space-between",
                    "padding": 15,
                    "paddingTop": 30,
                  }
                }
              >
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "fontWeight": "bold",
                    }
                  }
                >
                  Website
                </Text>
                <View>
                   
                   
                </View>
              </View>
              <View
                style={
                  Object {
                    "borderBottomColor": "grey",
                    "borderBottomWidth": 2,
                    "flex": 1,
                    "paddingBottom": 15,
                    "paddingTop": 15,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "justifyContent": "center",
                    }
                  }
                >
                  <View
                    accessibilityComponentType={undefined}
                    accessibilityLabel={undefined}
                    accessibilityTraits={undefined}
                    accessible={true}
                    collapsable={undefined}
                    hasTVPreferredFocus={undefined}
                    hitSlop={undefined}
                    isTVSelectable={true}
                    nativeID={undefined}
                    onLayout={undefined}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "opacity": 1,
                      }
                    }
                    testID={undefined}
                    tvParallaxProperties={undefined}
                  >
                    <View
                      style={
                        Array [
                          Object {
                            "alignItems": "center",
                            "backgroundColor": "white",
                            "flexDirection": "row",
                            "justifyContent": "center",
                          },
                          Array [],
                        ]
                      }
                    >
                      <View
                        style={
                          Object {
                            "alignItems": "center",
                            "justifyContent": "center",
                            "width": "15%",
                          }
                        }
                      >
                        <Text
                          accessible={true}
                          allowFontScaling={true}
                          ellipsizeMode="tail"
                        />
                      </View>
                      <View
                        style={
                          Object {
                            "alignItems": "flex-start",
                            "flexWrap": "wrap",
                            "justifyContent": "center",
                            "padding": 10,
                            "width": "70%",
                          }
                        }
                      >
                        <Text
                          accessible={true}
                          allowFontScaling={true}
                          ellipsizeMode="tail"
                        >
                           
                          asd
                        </Text>
                      </View>
                      <View
                        style={
                          Object {
                            "alignItems": "center",
                            "justifyContent": "center",
                            "width": "15%",
                          }
                        }
                      />
                    </View>
                  </View>
                </View>
              </View>
            </View>
            <View>
              <View
                style={
                  Object {
                    "alignItems": "center",
                    "borderBottomColor": "grey",
                    "borderBottomWidth": 1,
                    "flexDirection": "row",
                    "justifyContent": "space-between",
                    "padding": 15,
                    "paddingTop": 30,
                  }
                }
              >
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "fontWeight": "bold",
                    }
                  }
                >
                  Organizations
                </Text>
                <View>
                   
                   
                </View>
              </View>
              <View
                style={
                  Object {
                    "borderBottomColor": "grey",
                    "borderBottomWidth": 2,
                    "flex": 1,
                    "paddingBottom": 15,
                    "paddingTop": 15,
                  }
                }
              />
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "backgroundColor": "white",
            }
          }
        />
      </View>
    </RCTScrollView>
    <View
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
            "left": 0,
            "overflow": "hidden",
            "position": "absolute",
            "top": 0,
          },
          Object {
            "height": 65,
            "width": 750,
          },
        ]
      }
    >
      <View
        collapsable={undefined}
        style={
          Object {
            "backgroundColor": "#272727",
            "height": 65,
            "opacity": 0,
          }
        }
      >
        <View
          collapsable={undefined}
          style={
            Object {
              "transform": Array [
                Object {
                  "translateY": 65,
                },
              ],
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#272727",
                "height": 80,
                "justifyContent": "center",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "white",
                  "fontSize": 20,
                  "fontWeight": "bold",
                }
              }
            >
              asd
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "height": 30,
            "justifyContent": "center",
            "position": "absolute",
            "right": 10,
            "top": 25,
            "width": 30,
          }
        }
      >
        <View
          accessibilityComponentType={undefined}
          accessibilityLabel={undefined}
          accessibilityTraits={undefined}
          accessible={true}
          collapsable={undefined}
          hasTVPreferredFocus={undefined}
          hitSlop={undefined}
          isTVSelectable={true}
          nativeID={undefined}
          onLayout={undefined}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
            }
          }
          testID={undefined}
          tvParallaxProperties={undefined}
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
          />
        </View>
      </View>
    </View>
  </View>
</RCTSafeAreaView>
`;
